target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -94,9 +94,10 @@ define Device/ubnt_acb-isp
   UBNT_BOARD := ACB-ISP
   UBNT_CHIP := qca9533
   UBNT_TYPE := ACB
-  UBNT_VERSION := 6.0.0
-  IMAGES := sysupgrade.bin
+  UBNT_VERSION := 2.5.0
+  IMAGE/factory.bin := $$(IMAGE/sysupgrade.bin) | mkubntimage-split
```

Comment:
This will include "append-metadata" from IMAGE/sysupgrade.bin which shouldn't be required in a factory image. To replacing with the actual commands from sysupgrade.bin minus append-metadata.